llama_hub/singlestore/base.py

```diff
@@ -0,0 +1,72 @@
+"""SingleStore reader."""
+
+from typing import Any, Dict, List, Optional
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+from llama_index import download_loader, ListIndex
+
+import pymysql
```

Comment:
Please do the imports from within the class and raise import error with appropriate error messages. 

For example: https://github.com/emptycrown/llama-hub/blob/main/llama_hub/couchdb/base.py